DESCRIPTION

```diff
@@ -26,19 +26,20 @@ Imports:
     checkmate (>= 2.1.0),
     grid,
     htmltools (>= 0.5.3),
-    lifecycle (>= 0.2.0)
+    lifecycle (>= 0.2.0),
+    stringi (>= 1.7.12)
 Suggests:
     dplyr (>= 1.0.9),
```

Comment:
lets put it in imports and lose the require()

Reply:
but to_string uses stringi functions a bit, so imported them